.github/workflows/mvn-verify-check/populate-daily-cache.sh

```diff
@@ -14,27 +14,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -x
-max_retry=3; delay=30; i=1
+set -e
```

Comment:
would be good to standardize in the future if we enable debug output by default.

Reply:
Tested in [FAILED](https://github.com/YanxuanLiu/spark-rapids/actions/runs/12151847128/job/33887174202), the workflow failed with the bug.

Yes, for this script we disable debug output for clear log.